shared/copyq.iss

```diff
@@ -1,38 +1,49 @@
-﻿; 1. Open this file with Inno Setup with Unicode support and preprocessor.
+; 1. Open this file with Inno Setup with Unicode support and preprocessor.
 ; 2. Change "#defines" below (or see below how to use COPYQ_INNO_SETUP environment variable).
 ; 3. Compile "setup.exe".
 
 ; Path for output installation file
-#define Output "."
+#define Output         "."
+#define MyAppName      "CopyQ"
+#define MyAppNameMin   "copy"
```

Comment:
```suggestion
#define MyAppNameMin   "copyq"
```

Reply:
Tnahkls for the report about ypo error

Reply:
I updated the installer script with "MyAppCopyrightStartYear" and "MyAppCopyrighTEndYear variables".

Please check it. Thanks.